### DISCUSSION TITLE
Unable to Load Authentication Page

### DISCUSSION THREAD

**Discussion by Vleavell** [homebridge-simplisafe3/homebridge-simplisafe3#419] [General]
I am new to HomeBridge, so please bear with me. I have followed the instructions and just as others have reported, the authentication will not load. It just goes to a blank page. Then if you attempt to reload, you receive an error from SimpliSafe. 
Please let me know if this can be corrected as I am hoping to use this file. Thank you for the help! 

![Screenshot 2024-02-16 at 5 44 24 PM](https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/160294198/3527cd94-91f1-4e09-9527-219e6c148b58)
![Screenshot 2024-02-16 at 5 44 40 PM](https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/160294198/20f5e721-e912-4bd7-a16f-8613586cfd22)

**Comment by shamoon** [comment]
Please review the steps here carefully: https://github.com/homebridge-simplisafe3/homebridge-simplisafe3?tab=readme-ov-file#simplisafe-authentication

Particularly step iii

I just tested and it does work.

**Comment by Vleavell** [comment]
Thank you for the reply. I tried again, once I enter the login credentials it takes me to a blank screen. I tried Safari and Firefox. Neither work. 
<img width="670" alt="Screenshot 2024-02-16 at 10 12 50 PM" src="https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/160294198/e8b818f2-3284-4bdd-96b6-9ea59c519d77">
<img width="1434" alt="Screenshot 2024-02-16 at 10 16 37 PM" src="https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/160294198/5b869f85-0725-4d36-a00f-60326a463efa">
<img width="707" alt="Screenshot 2024-02-16 at 10 19 41 PM" src="https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/160294198/473d3679-5fee-4c34-8b5b-39b7b3e3c983">

**Reply by shamoon** [reply]
It really seems like you are not reading the instructions

<img width="846" alt="Screenshot 2024-02-16 at 10 35 10 PM" src="https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/4887959/2ebb47a2-25a0-465f-abee-c87beebc7067">

**Reply by Vleavell** [reply]
Thanks again. Very strange, I uninstalled the plugin, reinstalled the plugin and this time it worked. Or seems to be. Thank you again.

**Reply by shamoon** [reply]
Sweet.
